projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -353,7 +355,8 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
     return this.localTableConfig.dataMode === GoTableDataSource.server;
   }
 
-  private tableChangeOutcome(): void {
+  private tableChangeOutcome(event: string): void {
```

Comment:
should we emit this event with the tableChange, or do we need to have a param here?

Reply:
We actually don't need this param. This was from testing. Removed.